src/lib.rs

```diff
@@ -205,15 +206,20 @@ impl Header {
     }
 }
 
+pub struct Level<'a> {
+    pub bytes: &'a [u8],
+    pub uncompressed_length_bytes: u64,
```

Comment:
Before we expose this, let's make it more precisely consistent with the spec's `uncompressedByteLength`:
```suggestion
    pub uncompressed_byte_length: u64,
```